Documentation/flexvolume.md

```diff
@@ -46,7 +46,32 @@ Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path
 Rancher provides an easy way to configure kubelet. The FlexVolume flag will be shown later on in the [configuring the FlexVolume path](#configuring-the-flexvolume-path).
 It can be provided to the kubelet configuration template at deployment time or by using the `up to date` feature if Kubernetes is already deployed.
 
-Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
+Rancher deploys kubelet as a docker container, you need to mount the host's flexvolume path into the kubelet image as a volume,
+this can be done in the extra_binds section of the kubelet cluster config.
+
+Configure the rancher deployed kubelet by pdating the cluster.yml file kubetlet section:
+
+```yaml
+kubelet:
+ image: ""
+ extra_args:
+  volume-plugin-dir: /usr/libexec/kubernetes/kubelet-plugins/volume/exec
+  extra_binds:
+  - /usr/libexec/kubernetes/kubelet-plugins/volume/exec:/usr/libexec/kubernetes/kubelet-plugins/volume/exec
+```
+
+and run ```rke up```
+
+Configure the rook operator by updating the rook operator.yml file to include:
```

Comment:
I just noticed that this part isn't really needed as the flexvolume directory `/usr/libexec/kubernetes/kubelet-plugins/volume/exec` is the default.

If you want to mention the configuration of the operator (e.g., someone wants/has to use another flexvolume dir) nonetheless, you should remove the section and add a sentence or two saying that the `/usr/libexec/...` path is the default and nothing needs to be done, unless another path is chosen and then use this sentence (as can be found in the other sections):
```
Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
```
To not duplicate the Rook Ceph operator flexvolume configuration part.